WebPhapp/backend/server.js

```diff
@@ -258,7 +258,7 @@ Returns:
         prescriberID, dispenserID, cancelDate, drugName
     ]
 */
-app.get('/api/v1/prescriptions/:patientID', (req,res) => {
+app.get('/api/v1/prescriptions/:patientID', auth.checkAuth([Role.Patient, Role.Prescriber, Role.Government]), (req,res) => {
```

Comment:
Should dispensers be able to see this information? I think they should be able to.